susemanager/src/mgr_bootstrap_data.py

```diff
@@ -389,6 +389,40 @@
     "dmidecode"
 ]
 
+AMAZONLINUX2 = [
```

Comment:
This is required for the WebUI bootstrap, where `system-rpm-config` is required.